components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2107,6 +2110,7 @@ void RewardsServiceImpl::OnNotificationTimerFired() {
   GetReconcileStamp(
       base::Bind(&RewardsServiceImpl::MaybeShowAddFundsNotification,
         AsWeakPtr()));
+  ShouldShowLaunchNotification();
```

Comment:
why is this called here and not when ads is enabled (pref change)?

Reply:
discusses in slack